Index: app/src/main/res/layout/activity_stock_page.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"utf-8\"?>\r\n<LinearLayout xmlns:android=\"http://schemas.android.com/apk/res/android\"\r\n    xmlns:tools=\"http://schemas.android.com/tools\"\r\n    android:id=\"@+id/main\"\r\n    android:layout_width=\"match_parent\"\r\n    android:layout_height=\"match_parent\"\r\n    android:gravity=\"center\"\r\n    android:orientation=\"vertical\"\r\n    tools:context=\".StockPage\">\r\n\r\n\r\n\r\n    <Spinner\r\n        android:id=\"@+id/machLocker\"\r\n        android:layout_width=\"wrap_content\"\r\n        android:layout_height=\"wrap_content\" />\r\n\r\n    <Spinner\r\n        android:id=\"@+id/maintenceLocker\"\r\n        android:layout_width=\"wrap_content\"\r\n        android:layout_height=\"wrap_content\"/>\r\n\r\n</LinearLayout>
===================================================================
diff --git a/app/src/main/res/layout/activity_stock_page.xml b/app/src/main/res/layout/activity_stock_page.xml
--- a/app/src/main/res/layout/activity_stock_page.xml	(revision 37d6ea83826b7195e38510f2a5427f68255c11ce)
+++ b/app/src/main/res/layout/activity_stock_page.xml	(date 1771357451401)
@@ -9,15 +9,15 @@
     tools:context=".StockPage">
 
 
-
     <Spinner
-        android:id="@+id/machLocker"
+        android:id="@+id/macLocker"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content" />
+        android:layout_height="48dp"
+        />
 
     <Spinner
-        android:id="@+id/maintenceLocker"
+        android:id="@+id/maiLocker"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"/>
+        android:layout_height="48dp"/>
 
 </LinearLayout>
\ No newline at end of file
